test/integration/master/audit_test.go

```diff
@@ -0,0 +1,354 @@
+				expectNoError(t, err, "failed to patch configmap")
+
+				_, err = kubeclient.CoreV1().ConfigMaps(namespace).List(metav1.ListOptions{})
+				expectNoError(t, err, "failed to list config maps")
+
+				err = kubeclient.CoreV1().ConfigMaps(namespace).Delete(configMap.Name, &metav1.DeleteOptions{})
+				expectNoError(t, err, "failed to delete audit-configmap")
+			},
+			[]auditEvent{
+				{
+					auditinternal.LevelRequestResponse,
+					auditinternal.StageResponseComplete,
+					fmt.Sprintf("/api/v1/namespaces/%s/configmaps", namespace),
+					"create",
+					201,
+					auditTestUser,
+					"configmaps",
+					namespace,
+					true,
+					true,
+					"allow",
+				}, {
+					auditinternal.LevelRequestResponse,
+					auditinternal.StageResponseComplete,
+					fmt.Sprintf("/api/v1/namespaces/%s/configmaps/audit-configmap", namespace),
+					"get",
+					200,
+					auditTestUser,
+					"configmaps",
+					namespace,
+					false,
+					true,
+					"allow",
+				}, {
+					auditinternal.LevelRequestResponse,
+					auditinternal.StageResponseComplete,
+					fmt.Sprintf("/api/v1/namespaces/%s/configmaps", namespace),
+					"list",
+					200,
+					auditTestUser,
+					"configmaps",
+					namespace,
+					false,
+					true,
+					"allow",
+				}, {
+					auditinternal.LevelRequestResponse,
+					auditinternal.StageResponseStarted,
+					fmt.Sprintf("/api/v1/namespaces/%s/configmaps?timeoutSeconds=%d&watch=true", namespace, watchTestTimeout),
+					"watch",
+					200,
+					auditTestUser,
+					"configmaps",
+					namespace,
+					false,
+					false,
+					"allow",
+				}, {
+					auditinternal.LevelRequestResponse,
+					auditinternal.StageResponseComplete,
+					fmt.Sprintf("/api/v1/namespaces/%s/configmaps?timeoutSeconds=%d&watch=true", namespace, watchTestTimeout),
+					"watch",
+					200,
+					auditTestUser,
+					"configmaps",
+					namespace,
+					false,
+					false,
+					"allow",
+				}, {
+					auditinternal.LevelRequestResponse,
+					auditinternal.StageResponseComplete,
+					fmt.Sprintf("/api/v1/namespaces/%s/configmaps/audit-configmap", namespace),
+					"update",
+					200,
+					auditTestUser,
+					"configmaps",
+					namespace,
+					true,
+					true,
+					"allow",
+				}, {
+					auditinternal.LevelRequestResponse,
+					auditinternal.StageResponseComplete,
+					fmt.Sprintf("/api/v1/namespaces/%s/configmaps/audit-configmap", namespace),
+					"patch",
+					200,
+					auditTestUser,
+					"configmaps",
+					namespace,
+					true,
+					true,
+					"allow",
+				}, {
+					auditinternal.LevelRequestResponse,
+					auditinternal.StageResponseComplete,
+					fmt.Sprintf("/api/v1/namespaces/%s/configmaps/audit-configmap", namespace),
+					"delete",
+					200,
+					auditTestUser,
+					"configmaps",
+					namespace,
+					true,
+					true,
+					"allow",
+				},
+			},
+		},
+	}
+
+	expectedEvents := []auditEvent{}
+	for _, t := range testCases {
```

Comment:
What's the point of iterating over 1 statically defined element? Especially since you're merging all expectedEvents together anyway?

Reply:
Done